db/migrate/20230309115819_change_data_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ChangeDataService < ActiveRecord::Migration[7.0]
+  def change
+    change_table :data_services, bulk: true do |t|
+      t.text :endpoint_url
+      t.text :endpoint_description
+      t.integer :serves_data
```

Comment:
In the spec we said `serves_data` is a string but actually looking at Alastair's doc it looks like it's a kind of relationship to other data resources. Could you please remove it for now and I'll work with Alistair to understand what we expect to see here.

Reply:
I think it's clearer now.